susemanager/susemanager.changes

```diff
@@ -1,6 +1,7 @@
 - dmidecode does not exist on s390x (bsc#1145119)
 - move /usr/share/rhn/config-defaults to uyuni-base-common
 - Add missing bootstrap repo entries for Ubuntu repositories
+- Show help message when missing sub-command in  mgr-sync call (bsc#1134708)
```

Comment:
This is incorrect. New entries must be ON TOP. Otherwise this will not conflict with master and you will be changing an old sent of entries.

Please rebase from master and place the change on top.